dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/ConfigCenterConfigurationSourceImpl.java

```diff
@@ -87,7 +89,7 @@ public boolean isValidSource(Configuration localConfiguration) {
   public void init(Configuration localConfiguration) {
     configConverter = new ConfigConverter(ConfigCenterConfig.INSTANCE.getFileSources());
 
-    AddressManager kieAddressManager = configKieAddressManager();
+    kieAddressManager = configKieAddressManager();
```

Comment:
kieAddressManager only used once, do not need an instance property

Reply:
done